scripts/db_migrator.py

```diff
@@ -899,6 +899,17 @@ def common_migration_ops(self):
         else:
             log.log_notice("Asic Type: {}, Hwsku: {}".format(self.asic_type, self.hwsku))
 
+        route_table = self.appDB.get_table("ROUTE_TABLE")
```

Comment:
can you move this to a dedicated function.

Reply:
Sure, moved this logic to a method now.